care/emr/utils/batch_requests.py

```
@@ -87,8 +88,65 @@ def get_wsgi_request_object(curr_request, method, url, headers, body):
+        wsgi_request = get_wsgi_request_object(
+            parent_request,
+            request["method"],
+            request["url"],
+            request["headers"],
+            request["body"],
+        )
+        response = resp_generator(wsgi_request)
+        responses.append(response)
+        if (
+            request["reference_id"] in data_reference_required_id
+            and response["status_code"] < 300  # noqa PLR2004
```

Ruff (0.14.14)

[warning] Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)